### ISSUE TITLE
TodoMVC Example npm run build fails

### ISSUE THREAD

**Issue by Jakobud** [facebook/flux#253]
I was just starting out with the TodoMVC tutorial/example. I cloned down the `react-boilerplate` and then I copied the `flux/examples/todomvc/package.json` into the `react-boilerplate` directory. I then did `npm install` and then `npm run build` and I get this:

```
Error: Cannot find module '/Users/user/Sandbox/react-boilerplate' from '/Users/user/Sandbox/react-boilerplate'
    at /Users/user/Sandbox/react-boilerplate/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/Users/user/Sandbox/react-boilerplate/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/user/Sandbox/react-boilerplate/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/user/Sandbox/react-boilerplate/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:95:15)
```

What am I doing wrong? The tutorial says to make sure your `react-boilerplate` `package.json` is setup like the `package.json` from the example directory, so I just copied it over and overwrote it. Maybe that was incorrect to do? Any hints on this?

**Comment by lidorcg** [comment]
(+1)

**Comment by kyldvs** [comment]
Should be resolved now, re-open #288 if that's not the case.

**Comment by samuelemezzasalma** [comment]
I had the a similar error

> todomvc-flux@0.0.3 build /home/samuele/react-boilerplate
> browserify . -t [envify --NODE_ENV production] | uglifyjs -cm > js/bundle.min.js

sh: 1: cannot create js/bundle.min.js: Directory nonexistent
Error: Cannot find module '/home/samuele/react-boilerplate' from '/home/samuele/react-boilerplate'
    at /home/samuele/react-boilerplate/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/home/samuele/react-boilerplate/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/samuele/react-boilerplate/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /home/samuele/react-boilerplate/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)

npm ERR! Linux 3.19.0-33-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "build"
npm ERR! node v5.1.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! todomvc-flux@0.0.3 build: `browserify . -t [envify --NODE_ENV production] | uglifyjs -cm > js/bundle.min.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the todomvc-flux@0.0.3 build script 'browserify . -t [envify --NODE_ENV production] | uglifyjs -cm > js/bundle.min.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the todomvc-flux package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     browserify . -t [envify --NODE_ENV production] | uglifyjs -cm > js/bundle.min.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls todomvc-flux
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/samuele/react-boilerplate/npm-debug.log
